docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -55,13 +63,15 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
       <td colspan="2" class="yes">Yes</td>
       <td class="yes">Yes</td>
       <td class="yes">Yes</td>
+      <td class="yes">Yes</td>
     </tr>
     <tr>
       <td>Ignore issues and files,<br/>configure code patterns and file extensions,<br/>manage branches</td>
       <td>No</td>
       <td colspan="2" class="yes"><a href="#change-analysis-configuration">Configurable</a></td>
       <td colspan="2" class="yes"><a href="#change-analysis-configuration">Configurable</a></td>
       <td class="yes">Yes</td>
+      <td>Inherits original permission</td>
```

Comment:
The table only lists Codacy permissions, so this should be clear enough. Perhaps the problem is that we never mention in a header column that these are Codacy permissions. These tables are unfortunately not formatted properly.

I'll change this to "Inherits original Codacy role permission," but it would be pretty verbose. IMO, it's OK as it is.

Reply:
Yep, let's keep it as is for now and also revisit when we have the new billing manager role.